docs/api_reference.md

```diff
@@ -29,4 +30,13 @@ ws.addEventListener("message", (event) => {
     console.log("Message from server ", event.data);
 });
 ws.send(authRequest)
-```
\ No newline at end of file
+```
+
+
+The communication between peer WS and server is done with use of Protobuf format.
```

Comment:
It's the same sentence as in line 13, only with `Protobuf` instead of `JSON`, which can be confusing. We should probably clarify that we're referring to server notifications here, maybe something along the lines of "communication *from* server *to* peer WS".